codec-http2/src/main/java/io/netty/handler/codec/http2/HpackHeaderField.java

```diff
@@ -59,21 +60,16 @@ final int size() {
 
     @Override
     public final int hashCode() {
-        // TODO(nmittler): Netty's build rules require this. Probably need a better implementation.
-        return super.hashCode();
+        throw new UnsupportedOperationException();
```

Comment:
Isn’t it possible that the user calls this somehow ? Why not just leave it as it is ?

Reply:
I was unsure, but I made it throw because it would be unsafe to use this without knowing it's insensitive.

Reply:
Hmm still this seems wrong

Reply:
I can remove hashCode and equals.  WDYT?